docs/src/pages/docs/integrations/css.md

```diff
@@ -319,6 +321,29 @@ export default {
 };
 ```
 
+### Space Replacement
+
+Tokens and tokens groups can have whitespace in their names, but CSS variable names cannot contain whitespace. Leading and trailing whitespace will be removed during transformation and any whitespace in the middle of a group or token name will be replaced with the value of the `spaceReplacement` option.
```

Comment:
Great docs addition! Thanks for updating those. ❤️ (regardless of what the final API looks like, these are still very clear docs)